project-onboarding.md

```diff
@@ -29,3 +29,18 @@ Once the request has been made, the following steps will be taken:
 * If approved, a private channel will be created on the [Mac Admins Slack](https://macadmins.org) to facilitate communication and complete work around this process
 * The maintainer of the project will migrate the repo into the Mac Admins github org and the maintainer will be granted permissions to the repository
 * Once the project is migrated, signed and all other work is complete, the project will be made public and will be announced in [#macadmins-opensource](https://macadmins.slack.com/archives/C0224B1QUSK) as well as various social media outlets.
+
+## Maintenence
```

Comment:
~Maintenance~ Maintenance

Reply:
Ah nvm, already merged. Fixed in https://github.com/macadmins/macadmins.io/pull/8